implementations/ruby-json_schemer/Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM ruby:3-alpine as builder
 
 # unf_ext wants libc++ to compile itself
 RUN apk add --no-cache g++ make
+
+ARG JSON_SCHEMER_VERSION=2.3.0
+ENV JSON_SCHEMER_VERSION=$JSON_SCHEMER_VERSION
```

Comment:
The above two lines help us in dynamically setting the `JSON_SCHEMER` version during the docker / podman build process in the below manner:-

` docker build --build-arg JSON_SCHEMER_VERSION=0.2.17 .`

If no argument is provided then it defaults to the latest one (in this case the latest one is `v2.3.0` as of now).

Reply:
and here we set the environment variable with the `JSON_SCHEMER` version that was locked on.

Reply:
I think probably we should pick a cross-implementation convention here, so that someone can use the same thing across all versions. E.g. `IMPLEMENTATION_VERSION`?